admin/src/ducks/auth.js

```diff
@@ -66,15 +66,24 @@ export function signUp(email, password) {
 
         dispatch({
             type: SIGN_UP_SUCCESS,
-            payload: { user }
+            payload: {user}
         })
     }
 }
 
-/**
- * Init
- **/
-
-firebase.auth().onAuthStateChanged((user) => {
-    console.log('--- user', user)
-})
+export function checkAuth() {
+    return async (dispatch) => {
+        dispatch({
+            type: SIGN_IN_START
+        });
+
+        firebase.auth().onAuthStateChanged((user) => {
```

Comment:
Есть подозрение, что без перелогина второй раз checkAuth повесит приложение в SIGN_IN_START.
https://firebase.google.com/docs/reference/js/firebase.auth.Auth#onAuthStateChanged

> The observer will be triggered in the following scenarios:
> 
> When auth().onAuthStateChanged() is first called. It will trigger with the initial Auth state. If the user is returning from an auth().signInWithRedirect() operation, the observer will wait for that operation to resolve before initially triggering.
> When a new user signs.
> When an already signed in user signs out.